openquake/mbt/tools/fault_modeler/fault_modeling_utils.py

```diff
@@ -2786,27 +2866,32 @@ def calc_double_truncated_GR_mfd_from_fault_params(
         bin_width = fetch_param_val(fault_dict, 'bin_width', defaults=defaults,
                                     param_map=param_map)
 
-    bin_rates = rates_for_double_truncated_mfd(fault_area, seismic_slip_rate,
-                                               M_min, M_max,
-                                               b_value, bin_width,
-                                               rigidity=rigidity)
-
-    mfd = hz.mfd.EvenlyDiscretizedMFD(M_min + bin_width / 2.,
+    bin_mags, bin_rates = rates_for_double_truncated_mfd(fault_area, seismic_slip_rate,
+                                                         M_min, M_max,
+                                                         b_value, bin_width,
+                                                         rigidity=rigidity)
+    #
+    bin_rates_cli = get_rate_above_m_cli(bin_mags, bin_rates,
+                                          M_min, M_cli,
+                                          bin_width)
+    # Using rates from M_cli to M_max
+    mfd = hz.mfd.EvenlyDiscretizedMFD(M_cli + bin_width / 2.,
```

Comment:
For consistency, I would replace `M_cli + bin_width / 2.` with `with bin_mags[0]`

Reply:
bin_mags[0] = bin_mags[m_min], if m_min is different of m_cli the first magnitude bin I think will be wrong.

I solve this problem in this manner:
    bin_mags_cli, bin_rates_cli = get_rate_above_m_cli(bin_mags, bin_rates,
                                                                                         m_min, m_cli,
                                                                                         bin_width)
and then:
    mfd = hz.mfd.EvenlyDiscretizedMFD(bin_mags_cli[0],
                                                                   bin_width,
                                                                   bin_rates_cli)

Cheking bin_mags...
In config:
m_min = 4.0
m_cli = 6.5

Results:
bin_mags[0] =  4.05
bin_mags[m_cli+bin_width/2] =  6.55
bin_mags_cli[0] =  6.55